src/tags.rs

```diff
@@ -87,14 +91,24 @@ pub fn get_all(args: Args) -> Result<()> {
 
 /// Print the help message
 pub fn help(args: Args) -> Result<()> {
-    let help_string = "```
-?tag {key}
+    let help_string = if api::is_wg_and_teams(&args)? {
```

Comment:
We can also use this opportunity to note that tag creation is restricted as well. We can list all commands, and annotate which are role-locked.